src/snowflake/snowpark/dataframe_writer.py

```diff
@@ -109,11 +115,26 @@ def save_as_table(
             table_name if isinstance(table_name, str) else ".".join(table_name)
         )
         validate_object_name(full_table_name)
+        if create_temp_table:
+            warnings.warn(
+                "create_temp_table is deprecated, we still respect this parameter when it is True but "
```

Comment:
I think it would be better to make this 2 sentences, like this:

create_temp_table is deprecated. We still respect ...

Reply:
And end the sentence with a period.